diff --git a/src/mesh/MeshTypes.h b/src/mesh/MeshTypes.h
index 680926d3c..c6a3147b6 100644
--- a/src/mesh/MeshTypes.h
+++ b/src/mesh/MeshTypes.h
@@ -38,7 +38,7 @@ enum RxSource {
 #define HOP_MAX 7
 
 /// We normally just use max 3 hops for sending reliable messages
-#define HOP_RELIABLE 3
+#define HOP_RELIABLE 7
 
 // For old firmware or when falling back to flooding, there is no next-hop preference
 #define NO_NEXT_HOP_PREFERENCE 0
