diff --git a/userPrefs.jsonc b/userPrefs.jsonc
index 055f5927..a6ea1e6b 100644
--- a/userPrefs.jsonc
+++ b/userPrefs.jsonc
@@ -29,7 +29,7 @@
   // "USERPREFS_FIXED_GPS_LON": "2.294508368",
   // "USERPREFS_LORACONFIG_CHANNEL_NUM": "31",
   // "USERPREFS_LORACONFIG_MODEM_PRESET": "meshtastic_Config_LoRaConfig_ModemPreset_SHORT_FAST",
-  // "USERPREFS_SPLASH_TITLE": "DEFCONtastic",
+  "USERPREFS_SPLASH_TITLE": "Mining started",
   "USERPREFS_TZ_STRING": "tzplaceholder                                         "
   // "USERPREFS_USE_ADMIN_KEY_0": "{ 0xcd, 0xc0, 0xb4, 0x3c, 0x53, 0x24, 0xdf, 0x13, 0xca, 0x5a, 0xa6, 0x0c, 0x0d, 0xec, 0x85, 0x5a, 0x4c, 0xf6, 0x1a, 0x96, 0x04, 0x1a, 0x3e, 0xfc, 0xbb, 0x8e, 0x33, 0x71, 0xe5, 0xfc, 0xff, 0x3c }",
   // "USERPREFS_USE_ADMIN_KEY_1": "{}",
