diff --git a/src/mesh/Default.h b/src/mesh/Default.h
index d39886d1c..61bcde0de 100644
--- a/src/mesh/Default.h
+++ b/src/mesh/Default.h
@@ -19,7 +19,7 @@
 #define default_node_info_broadcast_secs 3 * 60 * 60
 #define default_neighbor_info_broadcast_secs 6 * 60 * 60
 #define min_node_info_broadcast_secs 60 * 60 // No regular broadcasts of more than once an hour
-#define min_neighbor_info_broadcast_secs 4 * 60 * 60
+#define min_neighbor_info_broadcast_secs 1 * 60 * 60
 
 #define default_mqtt_address "mqtt.meshtastic.org"
 #define default_mqtt_username "meshdev"
