diff --git a/src/mesh/MeshPacketQueue.cpp b/src/mesh/MeshPacketQueue.cpp
index d7ee6580..7c5969df 100644
--- a/src/mesh/MeshPacketQueue.cpp
+++ b/src/mesh/MeshPacketQueue.cpp
@@ -52,7 +52,7 @@ void fixPriority(meshtastic_MeshPacket *p)
                 // if text or admin, give high priority
             } else if (p->decoded.portnum == meshtastic_PortNum_TEXT_MESSAGE_APP ||
                        p->decoded.portnum == meshtastic_PortNum_ADMIN_APP) {
-                p->priority = meshtastic_MeshPacket_Priority_HIGH;
+                p->priority = meshtastic_MeshPacket_Priority_MSG;
                 // if it is a response, give higher priority to let it arrive early and stop the request being relayed
             } else if (p->decoded.request_id != 0) {
                 p->priority = meshtastic_MeshPacket_Priority_RESPONSE;
diff --git a/src/mesh/generated/meshtastic/mesh.pb.h b/src/mesh/generated/meshtastic/mesh.pb.h
index 5cd23c8e..de717d5d 100644
--- a/src/mesh/generated/meshtastic/mesh.pb.h
+++ b/src/mesh/generated/meshtastic/mesh.pb.h
@@ -422,6 +422,8 @@ typedef enum _meshtastic_MeshPacket_Priority {
     /* Ack/naks are sent with very high priority to ensure that retransmission
  stops as soon as possible */
     meshtastic_MeshPacket_Priority_ACK = 120,
+    /* Messages should have a slightly higher priority than Acks*/
+    meshtastic_MeshPacket_Priority_MSG = 121,
     /* TODO: REPLACE */
     meshtastic_MeshPacket_Priority_MAX = 127
 } meshtastic_MeshPacket_Priority;
