diff --git a/src/PowerFSM.cpp b/src/PowerFSM.cpp
index 4c4d203c..0e99e58f 100644
--- a/src/PowerFSM.cpp
+++ b/src/PowerFSM.cpp
@@ -42,7 +42,7 @@ static bool isPowered()
 
     // If we are not a router and we already have AC power go to POWER state after init, otherwise go to ON
     // We assume routers might be powered all the time, but from a low current (solar) source
-    bool isPowerSavingMode = config.power.is_power_saving || isRouter;
+    bool isPowerSavingMode = config.power.is_power_saving;
 
     /* To determine if we're externally powered, assumptions
         1) If we're powered up and there's no battery, we must be getting power externally. (because we'd be dead otherwise)
